deployability/modules/testing/tests/helpers/utils.py

```diff
@@ -2,12 +2,66 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
+import paramiko
+import re
 import yaml
+import logging
+import time
 
+from .logger.logger import logger
+
+
+paramiko_logger = logging.getLogger("paramiko")
+paramiko_logger.setLevel(logging.CRITICAL)
```

Comment:
Add spaces before the class definition
```suggestion
paramiko_logger.setLevel(logging.CRITICAL)


```